packages/devtools_server/lib/src/chrome.dart

```diff
@@ -55,35 +54,35 @@ class Chrome {
 
   /// Connects to an instance of Chrome with an open debug port.
   static Future<Chrome> fromExisting(int port) async =>
-      _connect(Chrome._(port, ChromeConnection('localhost', port)));
+      _connect(Chrome._(ChromeConnection('localhost', port), debugPort: port));
```

Comment:
we shouldn't create a ChromeConnection when there isn't a chrome remote debugging port.

Reply:
removed this code.